packages/web/src/components/errorMessage.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Box, Alert, AlertIcon, AlertDescription } from '@chakra-ui/react';
+export default function ErrorMessage({ message }) {
+  return (
+    <Box my={4}>
+      <Alert status="error" borderRadius={4}>
+        <AlertIcon />
+        <AlertDescription>{message}</AlertDescription>
+      </Alert>
+    </Box>
+  );
+}
```

Comment:
Make sure that you're creating `.tsx` and `.ts` files, Javascript files won't be picked up correctly.

`.ts` - for normal Typescript code
`.tsx` - for Typescript code that contains React code (`<Box`, `<Alert`, etc)

Reply:
Also it is recommended to do named exports over default exports. It helps with auto-complete in editors and also ensures that the same name is used throughout the application. 👍 

```suggestion
import React from 'react';
import { Box, Alert, AlertIcon, AlertDescription } from '@chakra-ui/react';
export function ErrorMessage({ message }) {
  return (
    <Box my={4}>
      <Alert status="error" borderRadius={4}>
        <AlertIcon />
        <AlertDescription>{message}</AlertDescription>
      </Alert>
    </Box>
  );
}
```